tests/pipe/test_pipe.cpp

```diff
@@ -299,22 +299,18 @@ SEQAN_DEFINE_TEST(test_pipe_test_sorter) {
     testSorter(MAX_SIZE);
 }
 
-template <typename TStringSet>
-inline void appendValues(TStringSet &stringSet, int numArgs, ...)
+template <typename TStringSet, typename T>
+inline void appendValues(TStringSet &stringSet, T t)
 {
-    va_list ap;
-    va_start(ap, numArgs);
-    for(int j = 0; j < numArgs; ++j)
-        appendValue(stringSet, va_arg(ap, char const *));
-    va_end(ap);
+    appendValue(stringSet, t);
 }
 
-template <typename TStringSet, typename TStrings>
-inline void appendValues(TStringSet &stringSet, TStrings const &strings)
+template <typename TStringSet, typename T, typename... Args>
+inline void appendValues(TStringSet &stringSet, T t, Args... args)
 {
-    typename Size<TStrings>::Type numStrings = length(strings);
-    for(int j = 0; j < numStrings; ++j)
-        appendValue(stringSet, strings[j]);
+    // use recursive variadic function, instead of va_list
+    appendValue(stringSet, t);
+    appendValues(stringSet, args...) ;
 }
```

Comment:
why where these functions changed? IIANM iterating is faster than recursion because recursive functions cannot be inlined...